buildenv/jenkins/variables/defaults.yml

```diff
@@ -166,6 +166,20 @@ freemarker:
 openjdk_reference_repo:
   openjdk_reference_repo: '/home/jenkins/openjdk_cache'
 #========================================#
+# BootJDK Default values
+#========================================#
+boot_jdk_default:
+  boot_jdk:
+    location:
+      all: '${HOME}/bootjdks'
+    version:
+      8: '8'
+      11: '11'
+      15: '14'
```

Comment:
We can use 15 to build 15, since it's already GAed.

Reply:
Updated